modules/tinymce/src/core/test/ts/browser/EditorRemoveTest.ts

```diff
@@ -29,6 +34,51 @@ UnitTest.asynctest('browser.tinymce.core.EditorRemoveTest', (success, failure) =
       cRemoveEditor
     ])),
 
+    Logger.t('remove editor after stylesheets load', Chain.asStep({}, [
+      Chain.op(() => {
+        // setup monitoring for uncaught errors
+        uncaughtErrors = [];
+        window.addEventListener('unhandledrejection', recordError);
+      }),
+      Chain.control(
+        McEditor.cFromHtml('<textarea></textarea>', {
+          ...settings,
+          setup: (editor: Editor) => {
+            editor.on('LoadContent', () => {
+              // Hook the function called when stylesheets are loaded
+              // so we can destroy the editor right after starting to load them.
+              const realLoadAll = editor.ui.styleSheetLoader.loadAll;
+              editor.ui.styleSheetLoader.loadAll = (...args) => {
+                realLoadAll.apply(editor.ui.styleSheetLoader, args);
+                editor.destroy();
```

Comment:
```suggestion
                editor.remove();
```
remove is our normal way to destroy an editor and `destroy` is just an alias for that, so we should probably use the common method here instead.

Reply:
Done